packages/react/use-overlay/src/OverlayController.tsx

```diff
@@ -16,24 +16,24 @@ export const OverlayController = forwardRef(function OverlayController(
   { overlayElement: OverlayElement, onExit }: Props,
   ref: Ref<OverlayControlRef>
 ) {
-  const [isOpen, setIsOpen] = useState(false);
+  const [isOpenOverlay, setIsOpenOverlay] = useState(false);
 
-  const handleClose = useCallback(() => setIsOpen(false), []);
+  const handleCloseOverlay = useCallback(() => setIsOpenOverlay(false), []);
```

Comment:
handleOverlayClose is more proper

We adhere event handler naming convention 

handle{targetName}{EventType}

Reply:
thank you!!